openlibrary/templates/books/custom_carousel.html

```diff
@@ -7,6 +7,20 @@
   $if slick_font not in ctx.links:
     $ ctx.links.append(slick_font)
 
+$def carousel_i18n_input():
+  $ show_carousel_i18n_strings=  {
+  $ "open": _("Read"),
+  $ "borrow_available": _("Borrow"),
+  $ "borrow_unavailable": _("Join Waitlist"),
+  $ "error": _("Not in Library")
```

Comment:
```suggestion
  $ "error": _("Not in Library"),
  $ "loading": _('Loading...')
```
Add "loading" string.